src/main/python/SentimentAnalyzer.py

```diff
@@ -0,0 +1,47 @@
+import pandas as pd
+# pip install --upgrade azure-ai-textanalytics
+from azure.ai.textanalytics import TextAnalyticsClient
+from azure.core.credentials import AzureKeyCredential
+
+_key = "adc96f39ac2a4918a1f462b5bf8674b8"
+_endpoint = "https://nbatwittersentiment.cognitiveservices.azure.com/"
+
+def _authenticate(key:str,endpoint:str):
+    # Authentication
+    ta_credential = AzureKeyCredential(key)
+    client = TextAnalyticsClient(
+        endpoint=endpoint,
+        credential=ta_credential,
+        api_version='TextAnalyticsApiVersion.V3_0')
+    return client
+
+def _formatresponse(responseDict) -> pd.DataFrame :
+    col_list = responseDict.keys()
+    cleanresponse = {'tweetID':responseDict.id,
+                    'positive':responseDict.confidence_scores.positive,
+                    'neutral':responseDict.confidence_scores.neutral,
+                    'negative':responseDict.confidence_scores.negative
+                    }
+    for idx, sentence in enumerate(responseDict.sentences):
+        cleanresponse ['tweet']= sentence.text
+    # sentiment_df = pd.DataFrame.from_dict(cleanresponse,orient='index')
+    sentiment_df = pd.DataFrame.from_records(cleanresponse,index=[0])
+    return sentiment_df
+
+def sentiment_analysis(tweet) -> pd.DataFrame:
+    """Uses Microsoft Cognitive Service Text Analytics 3.0 to analyze text
+    @param tweet[] list: a list of dictionary tweet objects
```

Comment:
This is a bit confusing. So the incoming variable "tweet" is a list of dictionaries where the dictionaries are {id, tweet}? Why would it need to be a list of dictionaries?

Reply:
this was poorly worded and laziness. The incoming variable can just be a dictionary that should include the tweeID, I wanted to preserve the functionality to submit multiple tweets at once. I can do this inside the method.